### ISSUE TITLE
Page About

### ISSUE THREAD

**Issue by Wilker-a** [agateau/pixelwheels#63]
the menu or the settings screen needs a page showing more info about the gake, along with useful links to places like this repository page.

the basic items to add should be

- the game and current version/build and release info (adaptable to number and custom build changes, too)
- the author(s)' name(s)
- maybe the description
- the licensing info, for license notices, and the license of this game with an option available to read it ingame. (because the game is under the General Public License and Creative Commons BY-SA, people should be able to know their rights and responsibilities \^_\^)
- useful links, such as a personal site (if you have any), the link to this repository, and other relevant things

i might be missing something, but i guess this should at least cover the basics.

what do you think? :3

**Comment by Wilker-a** [comment]
in another topic, i have a question about the "open for relicensing" that you mention in the readme file. i don't think that the GPL is compatible with licenses that can potentially make the game non-free, but it is compatible with things such as the AGPL though, which doesn't need many permissions to implement, so i don't think i understand what you meant by that...

**Comment by agateau** [comment]
Adding an about page is definitely a good idea :+1:.

What I mean with "open for relicensing" is that if an indie game developer wants to reuse a small GPL part of Pixel Wheels in their closed-source game, they should get in touch. If the request feels reasonable (and I reserve the right to decide if it is) and if all contributors to the affected files agree, then we can either dual-license the file under GPL v3+ and Apache 2.0, or if this code is actually generic enough, we could move it to the com.agateau package (which is all Apache 2.0) and relicense it under Apache 2.0.

To sum up, I don't mind helping indie game developers making a living, I believe we need more indie games.

**Comment by Wilker-a** [comment]
the [LGPL](https://choosealicense.com/licenses/lgpl-3.0/) can be an interesting license to use in the code :3

it keeps the copyleft for modifications of the code itself, but they can still use a closed source/non-free license if their own if they are to just implement it in their game as a library or interface, for example. \^\^

**Comment by Wilker-a** [comment]
[maybe going for GPL can actually be better than without copyleft](https://www.gnu.org/licenses/why-not-lgpl.html)? i think there should be more games where everyone can feel free to be curious and tinker with everything around them, and not just for modding. if we make more unique ideas and libraries under the GPL or another good copyleft license to make it free, this can bring in more space for free games to grow, as well as the lifespan of the games themselves. :3

what do you think? \^\^

###### this feels like just reiterating what is written in the article, although i still stand with what i say today...

**Comment by agateau** [comment]
I stand by my choice of licenses

**Comment by agateau** [comment]
This has been implemented in 0.17.0.
